### PR TITLE
feat: add nft usage command

### PR DESCRIPTION
**Pull request by AgwaB**

### CODE REVIEW

File: src/commands/nft/usage.ts [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  public static description = 'show nft api statistics';
+  public static examples = [`$ henesis nft:usage`];
+  public static flags = {};
+  public static args = [];
+
+  public async run(): Promise<void> {
+    try {
+      const [nftDailyStats] = await Promise.all([NFTHttp.dailyUsage()]);
+
+      if (nftDailyStats.length !== 0) {
```
Comment (zero1ne-dev) [inline src/commands/nft/usage.ts:30]: Wouldn't it be nice to show headlines and zero values only if there was no data?

File: src/rpc/nft.ts [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    trustedNodeBaseUrl: string,
+  ) {
+    this.server = server + '/nft/' + version + '/stats';
+    this.rpcVersion = version;
+    this.trustedNodeBaseUrl = trustedNodeBaseUrl;
+  }
+
+  public async dailyUsage(): Promise<NFTDailyStat[]> {
+    const clientId = getUserProperty('clientId');
+    const now = new Date().toISOString().split('T')[0];
```
Comment (zero1ne-dev) [inline src/rpc/nft.ts]: It seems dangerous to calculate the dates yourself, so why not use the dayjs library?
Please refer to my PR from brant/usage branch.

File: src/rpc/nft.ts [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    // json rpc db record is created at tomorrow. 1 day time difference exists now.
+    const startDate = getSpecificDayOfMonth(2).toISOString().split('T')[0];
```
Comment (zero1ne-dev) [inline src/rpc/nft.ts]: `getSpecificDayOfMonth` This seems to be replaced by the library too.

File: src/commands/nft/usage.ts [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+      return a + b.count;
+    }, 0);
+  }
+
+  private aggregateStats(filteredDailyStats: NFTDailyStat[]): NFTUsage[] {
+    let nftUsages: NFTUsage[] = [];
+
+    filteredDailyStats.forEach(nodeDailyStat => {
+      // json rpc db record is created at tomorrow. 1 day time difference exists now.
+      const utcDate = new Date(
```
Comment (zero1ne-dev) [inline src/commands/nft/usage.ts:65]: This seems to be replaced by the library too.
(See the comments below.)

File: src/rpc/nft.ts (hunk 2 above)
Comment (AgwaB) [reply src/rpc/nft.ts]: you mean `dayjs` library?

**Review by zero1ne-dev — APPROVED** [review]

**Pull request merged**